megamek/src/megamek/server/Server.java

```diff
@@ -30428,9 +30433,13 @@ private void receiveGameOptionsAux(Packet packet, int connId) {
      * Sends out the game victory event to all connections
      */
     private void transmitGameVictoryEventToAll() {
-        for (AbstractConnection conn : connections) {
-            send(conn.getId(), new Packet(Packet.COMMAND_GAME_VICTORY_EVENT));
+        if (connections == null) {
+            return;
         }
+
+        connections.stream()
+                .filter(Objects::nonNull)
+                .forEach(connection -> connection.send(new Packet(PacketCommand.GAME_VICTORY_EVENT)));
     }
```

Comment:
Just to make sure: It is intentional that this is no longer Server::send(...), instead it is now Connection::send? I looked at it quickly and it looked to me like it ends up being the same. But just making sure.

Reply:
It's identical, and easier to read this way. I've further changed it to use Server::send(Packet), which handles null checks and send to all available connections.